catalog/acm/setters.yaml

```diff
@@ -27,6 +27,8 @@ data:
   platform-project-id: platform-project-id
   # The URL for the repo for ACM (Config Sync)
   sync-repo: https://source.developers.google.com/p/{project_id}/r/{repo}
+  # The reference for the repo e.g. projects/${project}}/repos/${name}. Used to grant access to the ACM SA
+  sync-repo-ref: projects/project-id/repos/repo-name
```

Comment:
Cant we just use repo-project-id and repo-name to create interpolated setters that formats them into the URL an repo-ref as needed?

Reply:
I did think of that but if the repo is in a separate project than the cluster then it may be an issue. So this gives more flexibility in just providing the repo reference.